content/docs/details/scaffolder/index.md

```diff
@@ -866,11 +957,37 @@ steps:
       message: "Hello, World!"
 ```
 
+#### Outputs
+The `debug:log` action does not have any outputs.
+
 ### Other Actions
 You can find all of the actions available to your Backstage instance by visiting the following page from within Backstage:
 
 `https://<tenant-name>.roadie.so/create/actions`
 
+## Advanced
+### Calling an internal API
+If you need a scaffolder step to contact a custom authenticated service or any public API for that matter that is not currently supported by a built in action, you can do that using a combination of the `http:backstage:request` action and a [backstage proxy configuration](/docs/custom-plugins/authenticated-proxy/).
```

Comment:
This doesn't necessarily need to be an authenticated proxy. We should probably surface the simpler use case of using a standard proxy with a secret first

Reply:
That's true. I'll do that this morning.

Reply:
If you dont mind, Ill follow through the customer secrets and proxy configuration changes in a different commit.